### PR TITLE
feat(heading): merge `PPageTitle` and `PPanelTop` into `PHeading`

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [ ] Skipping review is not a problem (```style```, ```chore``` ONLY)
- [ ] Not that difficult

### Type of Change
- [x] New feature
- [ ] Bug fixes
- [x] Feature improvement
  - [x] These changes require a usage change
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [x] Updated Storybook documents for component
- [ ] Wrote test codes for composable, utils and etc.
- [x] Tested with console(if usages are changed) 

### Description
`PPageTitle` + `PPanelTop` => `PHeading`

스크린샷 2023-02-14 오후 2 11 51
스크린샷 2023-02-14 오후 2 11 58

merged into

스크린샷 2023-02-14 오후 2 12 18

### CODE REVIEW

File: src/data-display/heading/PHeading.vue [hunk 1]
```diff
@@ -52,13 +58,17 @@ interface Props {
...
     components: { PIconButton },
     props: {
         title: {
             type: String,
             default: 'Page',
         },
+        headingType: {
+            type: String as PropType<HeadingType>,
+            default: HEADING_TYPE.MAIN,
+        },
```
Comment (yuda110) [inline src/data-display/heading/PHeading.vue:71]: desc: `headingType` is added!

**Review by piggggggggy — APPROVED** [review]
LGTM!🫡🫡🫡🫡

**Review by sulmoJ — APPROVED** [review]
LGTM!
I just checked the code conventions!

**Pull request merged**